packages/eas-cli/src/build/utils/repository.ts

```diff
@@ -133,6 +133,7 @@ export async function makeProjectMetadataFileAsync(archivePath: string): Promise
     }
     throw e;
   }
+  clearTimeout(timer);
```

Comment:
Would it make sense to move it to `finally`? And remove the `catch`'s `clearTimeout` then?

Reply:
It would, on line 180 in `makeProjectTarballAsync` it is outside `finally` though.